tutorials/getting-started/pushing-assets.md

```diff
@@ -133,14 +143,20 @@ In your `my-theme` module, you'll include your assets in `lib/modules/my-theme/p
 
 As an example, with `karla.woff` in `lib/modules/my-theme/public/fonts` the @font-face definition could be as simple as:
 
+{% code-tabs %}
+{% code-tabs-item title="lib/modules/my-theme/public/fonts" %}
 ```css
 @font-face {
     font-family: 'Karla';
     src: url('/modules/my-theme/fonts/karla.woff') format('woff');
 }
 ```
+{% endcode-tabs-item %}
+{% endcode-tabs %}
 
-> Not working for you yet? Please check out the [complete, working example of a `theme` module in the apostrophe-samples project](https://github.com/apostrophecms/apostrophe-samples). This project is a great place to start if you need more examples of where to put your code "in context" so that everything works!
+{% hint style="working" %}
```

Comment:
What's a "working" hint?

Reply:
I'm interpreting the "working" hint as "is it working? this is why it might not be working". Currently, it's just showing up like an info hint in Gitbook, but I wanted to preserve the option to add the alternate style in there when I figure it out.